src/components/ListView/components/Item/components/Hover/Hover.js

```diff
@@ -0,0 +1,19 @@
+import styled, { css } from 'react-emotion';
+
+const baseStyles = ({ theme }) => css`
+  label: wrapper__item__hover;
+
+  display: none;
+  box-shadow: 0px 0px 0px 2px ${theme.colors.b500};
```

Comment:
Should this be a variable? Is this the box shadow that actually looks like a border radius?

Reply:
Yea, like a border.

And nice catch. The reason for this was that the `shadowBorder` util was not supporting bigger borders and I didn't want to change `circuit-ui` at the time. Now it is a good opportunity.